test/common/udp.js

```diff
@@ -1,21 +1,21 @@
 'use strict';
-const dgram = require('dgram');
+const { createSocket } = require('dgram');
 
 const options = { type: 'udp4', reusePort: true };
 
 function checkSupportReusePort() {
-  return new Promise((resolve, reject) => {
```

Comment:
Why is that not desirable in this case ?

Reply:
No particular reason. It was supposed to be a refactoring PR but I wouldn't see the value in opening a new pull-request for this changes. I can remove it, if requested. If not, it's a harmless change, and open to discussion in terms of readability...

Reply:
I prefer this syntax (you immediately see that a promise is returned), but I don't care enough to request changes

Reply:
This meaning the previous implementation or the change in this PR?

Reply:
Previous

Reply:
I see. Would you approve this PR if I changed these lines to the previous implementation?

Reply:
You don't need to change them